### DISCUSSION TITLE
Add the main network interface and subnet as an environment variables

### DISCUSSION THREAD

**Discussion by daniel-alomar** [netalertx/NetAlertX#555] [Ideas]
Since the main network interface is needed for a arp scan, and it is not always the same, it could be nice to have the chance to add as a environment variable the network interface to have correctly configured the arp scan. If we set also subnet we want to scan, Pi.Alert could begin with a valid scan, since ARP-SCAN is enabled by default.

**Comment by jokob-sk** [comment]
Thanks for the suggestion, let's see if someone else has any feedback on it. I don't want to overcomplicate an already pretty complex installation (overwriting a possibly user managed config file), but let's see what others say.
